src/main/java/org/jabref/gui/externalfiles/UnlinkedFilesDialogViewModel.java

```diff
@@ -145,9 +146,11 @@ public void startSearch() {
     }
 
     public void startImport() {
+        Path directory = Paths.get(this.getSearchDirectory().toString()); // Get the base directory
         List<Path> fileList = checkedFileListProperty.stream()
                                                      .map(item -> item.getValue().getPath())
                                                      .filter(path -> path.toFile().isFile())
+                                                     .map(path -> directory.toAbsolutePath().relativize(path.toAbsolutePath())) // Convert to relative path
```

Comment:
remoe the comment, it's clear from the code what is done.
Despite that I am not sure if you really need the second toAbsolutePath

Reply:
Also, the `directory` should already be an absolute path?